### PR TITLE
Forgotten Python integration in verrou package

### PR DESCRIPTION
**Pull request by HadrienG2**
The verrou package provides a couple of python-based tools, which I forgot to take into account. This PR fixes that.

### CODE REVIEW

File: var/spack/repos/builtin/packages/verrou/package.py [hunk 1]
```diff
@@ -28,7 +28,7 @@
 import sys
 
 
-class Verrou(AutotoolsPackage):
+class Verrou(AutotoolsPackage, PythonPackage):
```
Comment (HadrienG2) [inline var/spack/repos/builtin/packages/verrou/package.py]: I want to say that this autotools-based package provides python extensions. Is multiple inheritance the right way to do it? It seems to work currently, but I wouldn't want to rely on implementation-specific behaviour...

Comment (healther) [reply var/spack/repos/builtin/packages/verrou/package.py]: I'm pretty sure that is not the way to go. These base classes provide conflicting implementations for at least the `build` and `install` phases. 

If you only want to provide a python extension then you can add
```
extends('python')
```

But we should probably cover against this kind of implicit override... (Note: There is `CudaPackage` but it is not really a package in the sense of `AutotoolsPackage`).

Please correct me if I'm wrong @adamjstewart @alalazo

**Comment by HadrienG2** [comment]
Looks like some sort of network service timed out in the CI build.

File: var/spack/repos/builtin/packages/verrou/package.py (hunk 1 above)
Comment (HadrienG2) [reply var/spack/repos/builtin/packages/verrou/package.py]: Works for me. If there is no other comment, then I guess this PR is ready :smile:

**Comment by HadrienG2** [comment]
This PR is superseded by #9847

**Pull request closed without merging**